actor/todo/Makefile

```diff
@@ -0,0 +1,13 @@
+# examples/actor/todo
+
+PROJECT  = todo
+VERSION  = $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[] .version' | head -1)
+REVISION = 0
+# list of all contract claims for actor signing (space-separated)
+CLAIMS   = wasmcloud:httpserver wasmcloud:keyvalue
+# registry url for our actor
+REG_URL  = localhost:5000/v2/$(PROJECT):$(REVISION)
+# command to upload to registry (without last wasm parameter)
+PUSH_REG_CMD = wash reg push --insecure $(REG_URL)
+
+include ../../build/makefiles/actor.mk
```

Comment:
I think we also want a `make run` that will set up the capability provider etc.

I noticed that the order of setting up capability providers and links seems to matter (removing and recreating a cap provider doesn't seem to bring up its links reliably, so sometimes you need to bring the link down and up in order to make it work. Having this demonstrated in a script might help to isolate this problem)